src/hooks/useDownload.ts

```diff
@@ -39,14 +39,21 @@ async function internal<T>({ src, dst, headers, ephemeral, logger }: DownloadOpt
     return () => memo ?? (memo = hash_key(src))
   })()
   const mtime_entry = () => hash().join("mtime")
+  const etag_entry = () => hash().join("etag")
 
   const { numpty } = useFlags()
   dst ??= hash().join(src.path().basename())
   if (src.protocol === "file:") throw new Error()
 
-  if (!ephemeral && mtime_entry().isFile() && dst.isReadableFile()) {
+  if (!ephemeral && dst.isReadableFile()) {
+    
     headers ??= {}
-    headers["If-Modified-Since"] = await mtime_entry().read()
+    if(etag_entry().isFile()){
```

Comment:
our code format is space before the parameter after the `if`

Reply:
Fixed